omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+    * - 3.1
+      - from Jun 2009
+      - to June 2014
+      - Unsupported
+      - Unsupported
+      - `Link <http://legacy.python.org/dev/peps/pep-0375/>`__
+    * - 3.2
+      - from Feb 2011
+      - to Feb 2016
+      - Unsupported
+      - Unsupported
+      - `Link <http://legacy.python.org/dev/peps/pep-0392/>`__
+    * - 3.3
+      - from Sep 2012
+      - to Sep 2017
+      - Unsupported
+      - Unsupported
+      - `Link <http://legacy.python.org/dev/peps/pep-0398/>`__
+    * - 3.4
+      - from Mar 2014
+      - TBA
+      - Unsupported
+      - Unsupported
+      - `Link <http://legacy.python.org/dev/peps/pep-0429/>`__
+    * - 3.5
+      - TBA (probably Sep 2015)
+      - TBA
+      - Unsupported
+      - Unsupported
+      - `Link <https://www.python.org/dev/peps/pep-0478>`__
+
+Distribution support:
+
+.. tabularcolumns:: |l|l|L|L|L|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - CentOS/RHEL
+      - Fedora
+      - Ubuntu
+      - Debian
+      - Homebrew
+      - FreeBSD Ports
+    * - 2.5
+      - N/A
+      - N/A
+      - N/A
+      - 6.0
+      - N/A
+      - N/A
+    * - 2.6
+      - 6.x
+      - N/A
+      - 10.04
+      - 6.0, 7.0
+      - N/A
+      - Yes
+    * - 2.7
+      - 7.x
+      - 19, 20, 21
+      - 12.04, 13.04, 13.10, 14.04, 14.10, (15.04)
+      - 7.0, (8.0)
+      - Yes
+      - Yes
+    * - 3.0
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+    * - 3.1
+      - N/A
+      - N/A
+      - 10.04
+      - 6.0
+      - N/A
+      - N/A
+    * - 3.2
+      - N/A
+      - N/A
+      - 12.04
+      - 7.0
+      - N/A
+      - Yes
+    * - 3.3
+      - N/A
+      - 19, 20
+      - 13.04, 13.10
+      - N/A
+      - N/A
+      - Yes
+    * - 3.4
+      - N/A
+      - 21
+      - 14.04, 14.10, (15.04)
+      - (8.0)
+      - Yes
+      - Yes
+    * - 3.5
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+    * - Details
+      - 
+      - `Python 2 <https://apps.fedoraproject.org/packages/python>`__
+        `Python 3 <https://apps.fedoraproject.org/packages/python3>`__
+      - `Python 2 <http://packages.ubuntu.com/search?keywords=python2&searchon=names&suite=all&section=all>`__
+        `Python 3 <http://packages.ubuntu.com/search?keywords=python3&searchon=names&suite=all&section=all>`__
+      - `Python 2 <https://packages.debian.org/search?keywords=python2&searchon=sourcenames&suite=all&section=all>`__
+        `Python 3 <https://packages.debian.org/search?keywords=python3&searchon=sourcenames&suite=all&section=all>`__
+      - 
+      - 
+
+At the moment 2.7 support is present upstream for the foreseeable
+future; 3.x versions continue to be released and retired regularly in
+parallel.  The limiting factor will be distribution support for 2.7 as
+major packages are slowly switching to 3.x, and this might leave us in
+a mess if our python module dependencies are no longer available
```

Comment:
This can certainly be updated.  This is another area where we need to decide on a policy.  In this case it's probably quite simple (support 2.6/2.7 only for now), though there is a question of whether we should keep 2.6 support (which is tied into whether we support multiple CentOS major versions since retaining CentOS6 support forces 2.6 support).